earth_enterprise/src/fusion/autoingest/tools/getop.cpp

```diff
@@ -221,10 +221,14 @@ main(int argc, char *argv[])
             break;
           }
         }
+
         outline("");
         outline("Number of cached assets: %u", taskLists.num_assets_cached);
         outline("Number of cached asset versions: %u",
                 taskLists.num_assetversions_cached);
+        outline("Total size of cached assets: %lu", taskLists.asset_cache_size);
```

Comment:
After running it, I found I had to carefully count the number of digits to be sure of the size (`22631488` in this case).
It might be a nice quality of life change to make this human readable by adding a function like this to convert the number of bytes to a human readable form for output (found on Google, modified slightly but not tested):

`std::string GetReadableSize(double size/*in bytes*/) {
    int i = 0;
    const char* units[] = {"B", "kB", "MB", "GB", "TB", "PB", "EB", "ZB", "YB"};
    while (size > 1024) {
        size /= 1024;
        i++;
    }
    char buf[100];
    sprintf(buf, "%.*f %s", i, size, units[i]);
    return std::string(buf);
}`